### DISCUSSION TITLE
strange problem with montages/animations .

### DISCUSSION THREAD

**Discussion by assmikDuDuDuJ** [Sixze/ALS-Refactored#294] [Q&A]
There are two montages, the animations are different.
settings are the same, curves are the same.

When playing animation №1 - foot lock curve works.
When playing animation №2 - foot lock curve permanently = 1.
Even if in overlay animations, and montages foot lock = 0 or -1\2 etc.
how can i trace where this value comes from ?

**Comment by refikysg** [comment]
I have the same issue any solution yet ?

**Comment by xkamronx** [comment]
Post screenshots of each animation's window would help identify the problem.
